README.md

```diff
@@ -1,37 +1,76 @@
-# [Bedrock](https://roots.io/bedrock/)
-[![Build Status](https://travis-ci.org/roots/bedrock.svg)](https://travis-ci.org/roots/bedrock)
+# CNH Circle K Website
+CNH Circle K's website is built using WordPress. This project was generated using [Bedrock](https://roots.io/bedrock/).
 
-Bedrock is a modern WordPress stack that helps you get started with the best development tools and project structure.
+> Bedrock is a modern WordPress stack that helps you get started with the best development tools and project structure.
 
-Much of the philosophy behind Bedrock is inspired by the [Twelve-Factor App](http://12factor.net/) methodology including the [WordPress specific version](https://roots.io/twelve-factor-wordpress/).
+The project consists of two major components: the WordPress installation and the theme.
 
-## Features
+The theme is located in [`web/app/themes/cnh`](https://github.com/CNHCircleK/cnhcirclek.org/tree/master/web/app/themes/cnh). **Most development will be located here.**
 
-* Better folder structure
-* Dependency management with [Composer](http://getcomposer.org)
-* Easy WordPress configuration with environment specific files
-* Environment variables with [Dotenv](https://github.com/vlucas/phpdotenv)
-* Autoloader for mu-plugins (use regular plugins as mu-plugins)
-* Enhanced security (separated web root and secure passwords with [wp-password-bcrypt](https://github.com/roots/wp-password-bcrypt))
+## Requirements
 
-Use [Trellis](https://github.com/roots/trellis) for additional features:
+* PHP >= 5.5
+* MySQL
+* Composer - [Install](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx)
+* Node.js - [Install](https://nodejs.org/en/)
 
-* Easy development environments with [Vagrant](http://www.vagrantup.com/)
-* Easy server provisioning with [Ansible](http://www.ansible.com/) (Ubuntu 14.04, PHP 5.6 or HHVM, MariaDB)
-* One-command deploys
+## Getting Started
 
-See a complete working example in the [roots-example-project.com repo](https://github.com/roots/roots-example-project.com).
+### 0. Set up your environment
 
-## Requirements
+#### Composer
 
-* PHP >= 5.5
-* Composer - [Install](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx)
+Composer is a dependency manager for PHP.
+
+[Install](https://getcomposer.org/doc/00-intro.md#installation-linux-unix-osx)
+
+#### Node.js + npm
+
+We use Node.js and npm to build the frontend.
+
+[Install](https://nodejs.org/en/)
+
+
+#### Web Server
```

Comment:
I'm a little confused here - why is this section called "Web Server"? Isn't it covering development tools/environments? Is it because these tools are what allow us to spin up the server locally?

Reply:
It's covering your local web server - not the production web server.